scripts/check_configs.ts

```diff
@@ -596,9 +636,27 @@ async function main() {
         outcomeMap[network] = false;
       }
 
-      // transfer disabled tokens object check
-      const isTransferDisabledTokensValid = isValidTransferDisabledTokens(jsonData.transfer_disabled_tokens, tokens, network);
-      if (!isTransferDisabledTokensValid) outcomeMap[network] = false;
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      const { polynetwork = [], ibc = [], axelar = [] } = bridgesMap ?? {}
+      const polynetworkBridges = polynetwork.reduce((acc: string[], bridge) => {
+        if (bridge.enabled) acc.push(...bridge.bridgeAddresses)
+        return acc
+      }, [])
+
+      const axelarBridges = axelar.reduce((acc: string[], bridge) => {
+        if (bridge.enabled) acc.push(...bridge.bridgeAddresses)
```

Comment:
You need to use `bridge.bridgeAddress` instead of `bridge.bridgeAddresses`:
![image](https://github.com/user-attachments/assets/f38b3612-5469-4a28-815a-128fc6f1cfc8)